oak_runtime/src/introspect.rs

```diff
@@ -40,11 +40,11 @@ mod introspection_client {
     use hyper::header::CONTENT_ENCODING;
 
     fn static_file(
-        file: &str,
+        file: Vec<u8>,
```

Comment:
Does `&[u8]` not work here? `Vec` would cause the entire file to be re-allocated for each request, which is not ideal.

Reply:
Thanks for the heads up. :) `&'static [u8]` does work.

I had (mistakenly) assumed that accepting an array as an arg required typing out it's length (which would prohibit different files), but turns this works a-ok.

Reply:
- `[u8; 1024]` is an array of 1024 bytes, and would get passed as the entire 1024 bytes on the stack, by value (I think).
- `&[u8]` is a pointer to a byte slice, and would get passed as a pointer (a single word) on the stack.

Reply:
See https://doc.rust-lang.org/beta/rust-by-example/primitives/array.html

Reply:
makes sense :)